src/common.ts

```diff
@@ -296,6 +296,8 @@ export interface PlaceReview {
   language: string;
   /** the user's overall rating for this place. This is a whole number, ranging from 1 to 5. */
   rating: number;
+  /* The time since review in relative terms, for example '7 months ago' */
+  relative_time_description: string;
```

Comment:
This one should be removed. 

This field is specific to [Google Maps Platform JavaScript API ](https://developers.google.com/maps/documentation/javascript/reference/places-service#PlaceReview.relative_time_description) and not included in the response of the [HTTP libraries](https://developers.google.com/places/web-service/place-data-fields).

Reply:
Hey Justin, I was trying to move some frontend code to node serverside and I was actually seeing that data returned from the Client call, hence why I added them to remove the need for having to cast to any to get at them.

Perhaps an undocumented/misdocumeted feature?